packages/tokamak/contracts-bedrock/deploy-config/fork.devnetL1-template.json

```diff
@@ -0,0 +1,56 @@
+{
+  "l1ChainID": 900,
+  "l2ChainID": 901,
+  "l2BlockTime": 2,
+  "nativeTokenAddress": "0x2be5e8c109e2197D077D13A82dAead6a9b3433C5",
+  "maxSequencerDrift": 300,
+  "sequencerWindowSize": 200,
+  "channelTimeout": 120,
+  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000901",
+  "batchSenderAddress": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
+  "cliqueSignerAddress": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "l1UseClique": true,
+  "l1StartingBlockTag": "latest",
```

Comment:
Because we use the forked public network, we should get the latest block and start from this point. If you start from block 0, the forked public network doesn't have the base fee. So you can't generate the l2 files.